web/static/js/components/simulation/QuestionnaireSimulation.jsx

```diff
@@ -144,11 +147,23 @@ class QuestionnaireSimulation extends Component<Props, State> {
 
   render() {
     const { simulation } = this.state
+    const { router, projectId, questionnaireId } = this.props
     const simulationIsActive = simulation && simulation.simulationStatus == 'active'
+    const closeSimulationButton = (
+      <div>
+        <Tooltip text='Close Simulation'>
+          <a key='one' className='btn-floating btn-large waves-effect waves-light red right' style={{marginTop: '-6.1rem'}} href='#' onClick={() => router.push(routes.editQuestionnaire(projectId, questionnaireId))}>
```

Comment:
There is a `mtop` applied to other `btn-floating` that I guess fixes the margin-top also. Similar ammont, yet responsive it seems.

Reply:
Yes, but because we changed the default margin top of the whole container, the mtop wasn't the ammount we needed. I can create it in a class if that's better

Reply:
This page has a different margin top than the rest?

Reply:
Yes, indeed. The default for the rest of the site is 90px. In this is 60px. We followed the indesign mockup.
https://projects.invisionapp.com/d/main#/console/8374235/385278474/preview

Reply:
I see, there is no toolbar.... I would've preferred if the mtop class adapted to the layout with/without toolbar.

But ok.